profiles/multisite_drupal_communities/modules/custom/multisite_og_button/multisite_og_button.test

```diff
@@ -16,126 +24,133 @@ class OGButtonTestCase extends DrupalWebTestCase {
     );
   }
 
+  /**
+   * Initialize content types and users.
+   */
   public function setUp() {
-    // Enable any modules required for the test
+    // Enable any modules required for the test.
     parent::setUp(array(
-    	'multisite_og_button', 
-    	'og'
+        'multisite_og_button',
+        'og',
```

Comment:
These indentations are not correct, they are indenting 4 spaces instead of 2. This happens for many array declarations in the file.